### PR TITLE
Simplifications based on IntelliJ: Analyze --> Inspect code

### PR DESCRIPTION
**Pull request by HelgeStenstrom**
These simplifcations seem totally safe.

### CODE REVIEW

File: src/main/java/com/goxr3plus/streamplayer/stream/StreamPlayer.java [hunk 1]
```diff
@@ -436,7 +436,7 @@ private void createLine() throws LineUnavailableException, StreamPlayerException
 			// Calculate the Sample Size in bits
 			int nSampleSizeInBits = sourceFormat.getSampleSizeInBits();
 			if (sourceFormat.getEncoding() == AudioFormat.Encoding.ULAW || sourceFormat.getEncoding() == AudioFormat.Encoding.ALAW
-				|| nSampleSizeInBits <= 0 || nSampleSizeInBits != 8)
```
Comment (goxr3plus) [inline src/main/java/com/goxr3plus/streamplayer/stream/StreamPlayer.java]: I think this is incorrect here it removed nSampleSizeInBits <= 0 which is needed .

File: src/main/java/com/goxr3plus/streamplayer/stream/StreamPlayer.java [hunk 2]
```diff
@@ -1235,7 +1235,7 @@ public void setPan(final double fPan) {
 	 */
 	public void setGain(final double fGain) {
 		if (isPlaying() || isPaused() && hasControl(FloatControl.Type.MASTER_GAIN, gainControl))
-			gainControl.setValue((float) (20 * Math.log10(fGain != 0.0 ? fGain : 0.0000)));
```
Comment (goxr3plus) [inline src/main/java/com/goxr3plus/streamplayer/stream/StreamPlayer.java]: This is wrong here.

**Comment by HelgeStenstrom** [comment]
I was at a meating at work, and browsed the repo using an app on my phone. Then I clicked on a button that I hadn't clicked before, and suddenly the pull request was merged!

It wasn't my intention to merge, sorry about that.

File: src/main/java/com/goxr3plus/streamplayer/stream/StreamPlayer.java (hunk 1 above)
Comment (HelgeStenstrom) [reply src/main/java/com/goxr3plus/streamplayer/stream/StreamPlayer.java]: If (nSampleSizeInBits <= 0) is true, then also (nSampleSizeInBits != 8) is true.

Therefore (nSampleSizeInBits <= 0) is redundant.

**Comment by goxr3plus** [comment]
AHHAHAAHAHAHAHAHAHAH , that was so funny i spitted the water i was drinking ... don't worry .

I will revert the commit

**Comment by goxr3plus** [comment]
I reverted , some of your changes were good , some need to be changed . Please create again a new one and we will change accordingly some stuff.

File: src/main/java/com/goxr3plus/streamplayer/stream/StreamPlayer.java (hunk 2 above)
Comment (HelgeStenstrom) [reply src/main/java/com/goxr3plus/streamplayer/stream/StreamPlayer.java]: (fGain != 0.0 ? fGain : 0.0000) can be simplified to (fGain).

0.0 and 0.0000 is the same thing.

if fGain == 0d, the second alternative will be used, which is 0.0, which is fGain
if fGain != 02, the first alternative will be used, which is fGain.
Both paths leads to the value of fGain.

File: src/main/java/com/goxr3plus/streamplayer/stream/StreamPlayer.java (hunk 1 above)
Comment (goxr3plus) [reply src/main/java/com/goxr3plus/streamplayer/stream/StreamPlayer.java]: Hm , i didn't wrote that logic so i am not completely sure why it was there.

File: src/main/java/com/goxr3plus/streamplayer/stream/StreamPlayer.java (hunk 2 above)
Comment (goxr3plus) [reply src/main/java/com/goxr3plus/streamplayer/stream/StreamPlayer.java]: Hm i think you are right.

**Pull request merged**